main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala

```diff
@@ -211,10 +211,11 @@ object CompilerPerf {
    * The combine function uses to merge data from different runs.
    */
   def combine[T](xs: Seq[T])(implicit numeric: Numeric[T]): Double =
-    if (N <= 5)
+    if (N <= 4)
       numeric.toDouble(xs.last)
-    else
+    else {
       StatUtils.median(xs)
+    }
```

Comment:
What a weird style guide you follow. "Braces are mandatory only for the else clause"

Reply:
Fixed! I was messing around 👍